fixtures/index.js

```diff
@@ -815,6 +815,35 @@ module.exports.initMocks = function() {
     'Strict-Transport-Security',
     'max-age=15552000' ]);
 
+    nock('https://public-api.wordpress.com:443', {"encodedQueryParams":true})
+    .get('/rest/v1.1/sites/https%3A%2F%2Foddk.home.blog%2F')
```

Comment:
This should use mocked endpoint:

`.get('/rest/v1.1/sites/http%3A%2F%2F127.0.0.1%3A6000%2F')`

Reply:
If I run `npm run test` locally a lot of the tests fail, and Travis also was not satisfied with `.get('/rest/v1.1/sites/http%3A%2F%2F127.0.0.1%3A6000%2F')`

Reply:
@EvgeniiaVak I guess if you remove trailing slash it might succeed:

`.get('/rest/v1.1/sites/http%3A%2F%2F127.0.0.1%3A6000')`